src/sensai/vector_model.py

```diff
@@ -233,13 +62,27 @@ def __init__(self, checkInputColumns=True):
             e.g. in ensemble models.
         """
         super().__init__()
+        self._featureGenerator: Optional[FeatureGenerator] = None
+        self._inputTransformerChain = DataFrameTransformerChain()
+        self._outputTransformerChain = DataFrameTransformerChain()
         self._isFitted = False  # Note: this keeps track only of the actual model being fitted, not the pre/postprocessors
         self._predictedVariableNames: Optional[list] = None
         self._modelInputVariableNames: Optional[list] = None
         self._modelOutputVariableNames: Optional[list] = None
         self._targetTransformer: Optional[InvertibleDataFrameTransformer] = None
+        # TODO this was recently (December 6, 2020) introduced and will cause compatibility issues; should be removed
```

Comment:
not sure what you mean, `checkInputColumns` was there before the date you mention. I don't remember adding it myself (though I might well have, who knows ^^)

Reply:
No, it's older. I thought it was older but was fooled by the "blame" output. :)

Reply:
You probably added this when you introduced the ensemble models.
It struck me as odd because it's the only non-private property of VectorModel.